projects/kit/utils/date/update.ts

```diff
@@ -1,10 +1,12 @@
 import {TuiDay} from '@taiga-ui/cdk';
 
 export function tuiImmutableUpdateInputDateMulti(
-    value: readonly TuiDay[],
+    value: readonly TuiDay[] = [],
     day: TuiDay,
 ): readonly TuiDay[] {
-    return value.find(item => item.daySame(day))
-        ? value.filter(item => !item.daySame(day))
-        : value.concat(day);
+    const days = value ?? [];
```

Comment:
These changes are unnecessary since you added ` = []`

Reply:
fixed